pymeasure/instruments/hp/hp34401A.py

```diff
@@ -300,6 +303,25 @@ def init_trigger(self):
     )
 
     # System related commands
+    remote_local_state = Instrument.setting(
+        "SYST:%s",
+        """ A string property that controls the remote/local state of the
+        function generator. Valid values are: LOC<AL>, REM<OTE>, RWL<OCK>.
+        This setting can only be set. """,
+        validator=strict_discrete_set,
+        values=["LOC", "LOCAL", "REM", "REMOTE", "RWL", "RWLOCK"],
+        map_values=False,
+    )
```

Comment:
Is "RWL" relevant? If not, you could change it to a boolean "remote_control_enabled", which makes it a lot easier to understand: `remote_control_enabled = True`.

Reply:
Yes, for long measurements I find RWL very useful to provide protection against accidental reset of the remote control mode.

Reply:
Another idea as a **suggestion**: You have two boolean properties: `remote_control_enabled`, `remote_lock_enabled`.

Btw. you do not have to specify `map_value=True`.

Reply:
Ok, fixed to use boolean properties. But, wouldn't work without map_values set to true. But I guess you meant it for the first case.

Reply:
Sorry, I meant, `map_values=False` is the default value, therefore you do not have to specify it in the (outdated) code above.